src/olympia/migrations/989-replace-adminreview-with-addonreviewerflags.sql

```diff
@@ -0,0 +1,15 @@
+-- Make the old field nullable to prepare for its future removal.
+ALTER TABLE `addons` MODIFY COLUMN `adminreview` tinyint(1) NULL;
```

Comment:
Can you file the follow-up issue now so it's not forgotten.

Reply:
Done in #7000